test/Microsoft.IdentityModel.JsonWebTokens.Tests/JwtTokenUtilitiesTests.cs

```diff
@@ -13,6 +16,52 @@ namespace Microsoft.IdentityModel.JsonWebTokens.Tests
 {
     public class JwtTokenUtilitiesTests
     {
+        // Used for formatting a message for testing with one parameter.
+        private const string TestMessageOneParam = "This is the parameter: '{0}'.";
+
+        [Fact]
+        public void LogExceptionsWithStringJwe_JsonWebTokenHandler()
```

Comment:
>LogExceptionsWithStringJwe_JsonWebTokenHandler [](http://example.com/codeflow?start=20&length=46)

You don't validate the negative case, when ShowPII is set to false. This could be done on the same test.